### PR TITLE
Oceanwater 1120 light params switch lights

### PR DESCRIPTION
**Pull request by mogumbo**
Please see sibling PR: https://github.com/nasa/ow_simulator/pull/329

### CODE REVIEW

File: models/atacama_y1a/materials/scripts/ow_terrain.material [hunk 1]
```diff
@@ -53,14 +53,14 @@ fragment_program ow_terrain_frag glsl
     param_named albedoMap                      int 6
 
     // lander lights
-    param_named spotlightIntensityScale[0]     float 1.0
-    param_named spotlightIntensityScale[1]     float 1.0
+    // Control status of rover lights (spotlights)
```
Comment (AstroStucky) [inline models/atacama_y1a/materials/scripts/ow_terrain.material]:
```suggestion
    // Control status of lander lights (spotlights)
```

File: models/terminator/materials/scripts/ow_terrain.material [hunk 2]
```diff
@@ -52,14 +52,14 @@ fragment_program ow_terrain_frag glsl
     param_named detailNormalHeightMap          int 5
 
     // lander lights
-    param_named spotlightIntensityScale[0]     float 1.0
-    param_named spotlightIntensityScale[1]     float 1.0
+    // Control status of rover lights (spotlights)
```
Comment (AstroStucky) [inline models/terminator/materials/scripts/ow_terrain.material]:
```suggestion
    // Control status of lander lights (spotlights)
```

File: models/terminator_workspace/materials/scripts/ow_terrain.material [hunk 3]
```diff
@@ -52,14 +52,14 @@ fragment_program ow_terrain_frag glsl
     param_named detailNormalHeightMap          int 5
 
     // lander lights
-    param_named spotlightIntensityScale[0]     float 1.0
-    param_named spotlightIntensityScale[1]     float 1.0
+    // Control status of rover lights (spotlights)
```
Comment (AstroStucky) [inline models/terminator_workspace/materials/scripts/ow_terrain.material]:
```suggestion
    // Control status of lander lights (spotlights)
```

File: models/test_dem/materials/scripts/ow_terrain.material [hunk 4]
```diff
@@ -52,14 +52,14 @@ fragment_program ow_terrain_frag glsl
     param_named detailNormalHeightMap          int 5
 
     // lander lights
-    param_named spotlightIntensityScale[0]     float 1.0
-    param_named spotlightIntensityScale[1]     float 1.0
+    // Control status of rover lights (spotlights)
```
Comment (AstroStucky) [inline models/test_dem/materials/scripts/ow_terrain.material]:
```suggestion
    // Control status of lander lights (spotlights)
```

**Review by AstroStucky — APPROVED** [review]

File: models/atacama_y1a/materials/scripts/ow_terrain.material (hunk 1 above)
Comment (mogumbo) [reply models/atacama_y1a/materials/scripts/ow_terrain.material]: fixed

File: models/terminator/materials/scripts/ow_terrain.material (hunk 2 above)
Comment (mogumbo) [reply models/terminator/materials/scripts/ow_terrain.material]: fixed

File: models/terminator_workspace/materials/scripts/ow_terrain.material (hunk 3 above)
Comment (mogumbo) [reply models/terminator_workspace/materials/scripts/ow_terrain.material]: fixed

File: models/test_dem/materials/scripts/ow_terrain.material (hunk 4 above)
Comment (mogumbo) [reply models/test_dem/materials/scripts/ow_terrain.material]: fixed

**Review by kmdalal — APPROVED** [review]
My review was light as I'm not familiar with these files and their formats.

**Pull request merged**